trampoline/src/main/java/org/ernest/applications/trampoline/controller/SettingsController.java

```diff
@@ -126,4 +127,16 @@ public void checkoutAndPullAndRestart(@RequestParam(value="id") String id, @Requ
 		List<String> intancesIds = ecosystemManager.getEcosystem().getInstances().stream().filter(i-> i.getMicroserviceId().equals(id)).map(Instance::getId).collect(Collectors.toList());
 		intancesIds.forEach(intancesId-> ecosystemManager.restartInstance(intancesId));
 	}
+
+	@RequestMapping(value= "/git/save", method = RequestMethod.POST)
```

Comment:
Maybe this should be "/git/config/save" to mimic how git has the `git config user.name`/`git config user.password` syntax and make it more clear what this endpoint does?